### PR TITLE
[Workers] Make an example for environment variable with multiple environment

### PR DESCRIPTION
**Pull request by allevo**
I spent 10 minutes merging some information collected on the documentation to understand how to set different environment variables for different environments.

This PR could help.

### CODE REVIEW

File: content/workers/configuration/environment-variables.md [hunk 1]
```diff
@@ -49,6 +49,24 @@ export default {
 }
 ```
 
+It is possible to define environment variables for different environments as below:
```
Comment (deadlypants1973) [inline content/workers/configuration/environment-variables.md]:
```suggestion
`vars` is a non-inheritable key. [Non-inheritable keys](/workers/wrangler/configuration/#non-inheritable-keys) are configurable at the top-level, but cannot be inherited by environments and must be specified for each environment.

To define environment variables for different environments, refer to the example below:

```

**Review by deadlypants1973 — APPROVED** [review]
PCX

File: content/workers/configuration/environment-variables.md [hunk 2]
```diff
@@ -49,6 +49,24 @@ export default {
...
+```toml
+---
+filename: wrangler.toml
+---
+name = "my-worker-dev"
```
Comment (penalosa) [inline content/workers/configuration/environment-variables.md:60]:
```suggestion
name = "my-worker"
```

File: content/workers/configuration/environment-variables.md [hunk 3]
```diff
@@ -49,6 +49,27 @@ export default {
...
+
+To define environment variables for different environments, refer to the example below:
+
+```toml
+---
+filename: wrangler.toml
+---
+name = "my-worker-dev"
+
+[[env.staging.vars]]
```
Comment (allevo) [inline content/workers/configuration/environment-variables.md]:
```suggestion
[env.staging.vars]
```

File: content/workers/configuration/environment-variables.md [hunk 4]
```diff
@@ -49,6 +49,27 @@ export default {
...
+API_HOST = "staging.example.com"
+API_ACCOUNT_ID = "staging_example_user"
+SERVICE_X_DATA = { URL = "service-x-api.dev.example", MY_ID = 123 }
+
+[[env.production.vars]]
```
Comment (allevo) [inline content/workers/configuration/environment-variables.md]:
```suggestion
[env.production.vars]
```

**Review by allevo — COMMENTED** [review]
Wrong toml

**Review by WalshyDev — APPROVED** [review]

**Review by deadlypants1973 — APPROVED** [review]
PCX

**Pull request merged**